core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -513,8 +513,8 @@ private[spark] object UIUtils extends Logging {
     param
   }
 
-  def getTimeZoneOffset() : Int =
-    TimeZone.getDefault().getOffset(System.currentTimeMillis()) / 1000 / 60
+  def getTimeZoneOffset(): Int =
+    TimeZone.getDefault.getOffset(System.currentTimeMillis()) / 1000 / 60
```

Comment:
I also wouldn't remove the parens here (though the convention was always to use parens when calling no-arg Java methods?) or elsewhere. That really isn't done consistently at all anyway.